File: src/test/java/io/mfedirko/fixture/ContactForms.java
package io.mfedirko.fixture;

import io.mfedirko.contactme.ContactForm;

public class ContactForms {
    public static ContactForm.ContactFormBuilder aContactForm() {
        return ContactForm.builder()
                .fullName("John Doe")
                .email("johndoe@gmail.com")
                .messageBody("my message")
                .recaptcha("validcaptcha");
    }
}

File: src/test/kotlin/io/mfedirko/fixture/ContactForms.kt
package io.mfedirko.fixture

import io.mfedirko.contactme.ContactForm

object ContactForms {
    fun aContactForm(): ContactForm {
        return ContactForm(
            fullName = "John Doe",
            email = "johndoe@gmail.com",
            messageBody = "my message",
            recaptcha = "validcaptcha"
        )
    }
}